docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.8'
+
+# CloudSpawner only reads from the first element of lists
+services:
+  mini_tree:
+    image: summadev/summa-aggregation-mini-tree:latest
+    ports: 
+      - published: 4000
+        target: 4000
+    deploy:
+      replicas: 2
```

Comment:
Why 2 replicas, is this for local testing only?

Reply:
Yes, the number of replicas as an example

Reply:
Yes, It is used when specified `service_info` to the CloudSpawner.